.gitignore

```diff
@@ -10,8 +10,8 @@ tags
 doc/api.xml
 tmp/
 test/tmp*/
-iojs
-iojs_g
```

Comment:
I'd prefer to still gitignore these for now. Will get really silly going back and forth.